src/components/LaunchFilter/actions/actionTypesTemplates.ts

```diff
@@ -0,0 +1,5 @@
+export const enum LaunchActions {
```

Comment:
I think it better to rename the file on actionTypesLaunch
it will be more clear

Reply:
changed to launchActionTypes